target/linux/ath79/dts/qca9558_engenius_emr3000-v2.dts

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "EnGenius EMR3000 V2";
+	compatible = "engenius,emr3000-v2", "qca,qca9558";
+
+	aliases {
+		led-boot = &led_status_white;
+		led-failsafe = &led_status_white;
+		led-running = &led_status_white;
+		led-upgrade = &led_status_white;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_white: status_white {
+			label = "white:status";
+			gpios = <&gpio 18 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		status_red {
+			label = "red:status";
+			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
+		};
+
+		status_amber {
+			label = "amber:status";
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+		};
+
+		bluetooth {
+			label = "blue:bluetooth";
+			gpios = <&gpio 19 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy0: ethernet-phy@0 {
+		reg = <1>;
+		phy-mode = "rgmii";
+	};
+
+	phy1: ethernet-phy@1 {
+		reg = <2>;
+		phy-mode = "sgmii";
+		at803x-override-sgmii-link-check;
+	};
+};
```

Comment:
the label "phyX" and the node name "@x" should match the `reg` property
also, this is not where `phy-mode` goes, so remove
also, I understand that these PHY are AR8033 and AR8035, which are known to have issues with the EEE feature
so that can be disabled

for example:

```
	phy1: ethernet-phy@1 {
		reg = <1>;
		eee-broken-100tx;
		eee-broken-1000t;
	};
```